### PR TITLE
updated front matter with new summaries, added fields for shortname and acronym

### PR DESCRIPTION
**Pull request by nick-mon1**
Update to `/communities` with new summaries for the CoP.
Adds front matter fields for CoP:

- `dg_shortname`
- `dg_acronym`

**Before**
Screen Shot 2022-07-13 at 1 03 36 PM

**After**
Screen Shot 2022-07-13 at 1 03 27 PM

### CODE REVIEW

**Review by mejiaj — COMMENTED** [review]
@nick-mon1 looks good, just a few comments/questions.

Where is the `dg_acronym` used? I only see a acronym for `Communicators`, the rest are full words.

**Comment by nick-mon1** [comment]
Merged with new logo updates

Screen Shot 2022-07-14 at 5 10 47 PM

File: content/communities/multilingual.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 slug: multilingual
 date: 2014-02-21 4:58:10 -0400
 title: "Multilingual "
```
Comment (mejiaj) [inline content/communities/multilingual.md:7]: Is the extra space in title necessary?

**Review by mejiaj — CHANGES_REQUESTED** [review]
Small note on extra space in a title, otherwise LGTM

File: content/communities/multilingual.md (hunk 1 above)
Comment (nick-mon1) [reply content/communities/multilingual.md]: Good catch, I have removed the extra white space

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**